store/localcachelayer/layer.go

```diff
@@ -227,6 +239,9 @@ func (s *LocalCacheStore) doClearCacheCluster(cache *utils.Cache) {
 
 func (s *LocalCacheStore) Invalidate() {
 	s.doClearCacheCluster(s.reactionCache)
+	s.doClearCacheCluster(s.schemeCache)
```

Comment:
It is not very clear to me why we are clearing schemeCache and roleCache too. Was it just a bug, and you are fixing it here ?

Reply:
Yes